File: models/record.js
class Record{
    constructor(id,patientName,patientId,medications,allergies,history,hospitals){
        this.id = id,
        this.patientName = patientName,
        this.patientId = patientId,
        this.medications = medications,
        this.allergies = allergies,
        this.history = history,
        this.hospitals = hospitals
    }
}
module.exports = Record;

File: gui/lib/models/record.dart
import 'dart:convert';

import 'package:GPV1/Screens/record.dart';
import 'package:http/http.dart' as http;

class Record {
  final String history;
  final String allergies;
  final String hospitals;
  final String medications;

  Record({this.history, this.allergies, this.hospitals, this.medications});

  factory Record.fromJson(Map<String, dynamic> json) {

    return Record(
        history: json['history'],
        allergies: json['allergies'],
        hospitals: json['hospitals'],
        medications: json['medications']);
  }
}
